msbuild/Xamarin.MacDev.Tasks/Tasks/ComputeRemoteGeneratorPropertiesTaskBase.cs

```diff
@@ -146,5 +151,20 @@ void ComputeProperties ()
 				}
 			}
 		}
+
+		public void Cancel ()
+		{
+			if (ShouldExecuteRemotely ())
+				BuildConnection.CancelAsync (BuildEngine4).Wait ();
+		}
+
+		public bool ShouldCopyToBuildServer (ITaskItem item)
+		{
+			return false;
+		}
+
+		public bool ShouldCreateOutputFile (ITaskItem item) => false;
```

Comment:
Why not use the same code in both?  I mean, either use => or  {} in both.

Reply:
I'm just moving code here, and I figured it would be easier to remove if the code stayed the same.